.github/CODEOWNERS

```diff
@@ -0,0 +1,71 @@
+# Default match everything to the CORE team.
+# This means that items can be intentionally excluded below by using a match pattern without a group
+* @pachyderm/CORE
+
+# Docs
+/doc/ @pachyderm/DOC
+
+# etc
+/etc/ @pachyderm/BRE @pachyderm/CORE
+
+# Helm
+/etc/helm/ @jrockway @seslattery @robert-uhl @armaanv @chainlink
+
+# Services with individual owners
+/src/auth/               @acohen4 @jrockway
+/src/server/auth/        @acohen4 @jrockway
+
+/src/debug/              @jrockway @brycemcanally
+/src/server/debug/       @jrockway @brycemcanally
+
+/src/enterprise/         @acohen4
+/src/server/enterprise/  @acohen4
+
+/src/identity/           @jrockway @acohen4
+/src/server/identity/    @jrockway @acohen4
+
+/src/license/            @acohen4
+/src/server/license/     @acohen4
+
+/src/pfs/                @pachyderm/PFS
+/src/server/pfs/         @pachyderm/PFS
+
+/src/pps/                @brycemcanally @acohen4
+/src/server/pps/         @brycemcanally @acohen4
+
+/src/task/               @brycemcanally
+
+# Integrations
+/src/integrations/                         @pachyderm/INT
+/src/integrations/connectors/snowflake/    @robert-uhl
```

Comment:
This should be `@pachyderm/INT` and `@albscui`

Reply:
Line 40, 39 or both?